app/api/feed/json/route.ts

```diff
@@ -0,0 +1,14 @@
+import { NextResponse } from "next/server";
```

Comment:
These routes are currently set-up to be

* https://measured.co/api/feed/atom
* https://measured.co/api/feed/json
* https://measured.co/api/feed/rss

This doesn't align with the `<link>` tags you've added in `layout.tsx` (see separate comment), but I think the `layout.tsx` is probably correct. I imagine we probably want something like this, but it's been a long time since I've done this so might be off the mark:

* https://measured.co/feed.atom
* https://measured.co/feed.json
* https://measured.co/feed.xml
* https://measured.co/feed.rss (redirect to `.xml`, or just serve same content)
* https://measured.co/feed (redirect to `.xml`, or just serve same content) 

Alternatively, it could be good to put these under `/blog`, i.e. https://measured.co/blog/feed.atom.

Reply:
There's a rewrite in next.config dude

Reply:
Also I'm using best practise described here https://blog.jim-nielsen.com/2021/feed-urls/

Reply:
Oops!